resolwe_bio/processes/alignment/bwa.yml

```diff
@@ -353,7 +373,13 @@
       re-progress 0.5
 
       echo "COMPRESSING sam to sorted bam:"
-      samtools view -bS "${NAME}_align_unsorted.sam" | samtools sort -o "${NAME}.bam" -
+      # Samtools sort may require 4-5 GB RAM per thread, so the CPU
+      # limit for this command is set to 3
+      samtools view \
```

Comment:
Would this be more readable? Putting `samtools sort` into a new line.

```
      samtools view \
        -@ {{ requirements.resources.cores }} \
        -bS "${NAME}_align_unsorted.sam" | \
      samtools sort \
        -@ {{ [3, requirements.resources.cores]|min }} \
        -o "${NAME}.bam" -
```

If you think this is a good idea, see also `alignment-bwa-aln` process.